### DISCUSSION TITLE
Read Only connection: Allow to Format DAX Expression feature

### DISCUSSION THREAD

**Discussion by rem-bou** [TabularEditor/TabularEditor3#1031] [Ideas]
Hi team,

On TE 3.4.2, I just noticed that when using a read only connection to a model publish, we can not use the "Format DAX" as it is greyed out.
Would it be possible to provide this option in the future? 
This would make it easier to understand DAX expression when trying to figure out a model. Of course, it would not be possible to save the update to the model, but it would make the code much more readable.

**Comment by mlonsk** [comment]
Hi @rem-bou 
I will look into the feasibility of this and get back to you.
Thank you for your suggestion

**Comment by odinism** [comment]
I experiencing the same issue on verison 3.6.0
![Snipaste_2023-04-28_16-40-28](https://user-images.githubusercontent.com/39026877/235099458-e99767a9-22fb-4cc8-ac0c-bf29c799834e.png)

**Comment by DBojsen** [comment]
Hi @rem-bou & @odinism 

This is still on our backlog.  We have no ETA at this time.

BR David - TE3 Support
